environments/variable.tf

```diff
@@ -15,3 +15,5 @@ variable "vnet_name" {}
 variable "vnet_rg" {}
 variable "product" {}
 variable "builtFrom" {}
+variable "traffic_manager_endpoints" {}
```

Comment:
won't this fail because you haven't defined any?

Reply:
It shouldn't fail - It should do nothing. I'll add the configuration shortly but I need the pipeline first so figured I'd add the Terraform at the same time.